fix(learn): Cross browser testing - correct misleading text

ShamarYarde opened the pull request:
Previous text implied that `driver.executeScript("lambda-status=passed");` and `driver.executeScript("lambda-status=failed");` should be ran inside a Bash terminal shell instead of inside added inside the `lambdatest_google_test.js` demo code.

Running either as a command in a Bash terminal shell results in the following:

```js
bash: syntax error near unexpected token `"lambda-status=passed"'
```

if 

```js
driver.executeScript("lambda-status=passed");
```

is ran, and 

```js
bash: syntax error near unexpected token `"lambda-status=failed"'
```

if 

```js
driver.executeScript("lambda-status=failed");
```

is ran.


Adding either within the `lambdatest_google_test.js` demo code produces respective results on the LambdaTest Automation dashboard. Also, it looks like these should be inside `if` and `else` blocks of code.

<!-- 🙌 Thanks for contributing to MDN Web Docs. Adding details below will help us to merge your PR faster. -->

### Description

<!-- ✍️ Summarize your changes in one or two sentences -->
Changed explanatory text to indicate where code should be placed.
### Motivation

<!-- ❓ Why are you making these changes and how do they help readers? -->
To allow readers to understand how to mark a test an automation test's status as passed or failed. Also, to remove confusion on where to look to make changes in the [Your own remote server](https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Cross_browser_testing/Your_own_automation_environment#your_own_remote_server) section.

### Additional details

<!-- 🔗 Link to release notes, vendor docs, bug trackers, source control, or other places providing more context -->

### Related issues and pull requests

<!-- 🔨 If this fully resolves a GitHub issue, use "Fixes #123" -->
<!-- 👉 Highlight related pull requests using "Relates to #123" -->
<!-- ❗ If another pull request should be merged first, use "**Depends on:** #123" -->
"Fixes #31074"

<!-- 👷‍♀️ After submitting, go to the "Checks" tab of your PR for the build status -->

ShamarYarde committed:
corrected misleading text

Previous text implied that `driver.executeScript("lambda-status=passed");` and `driver.executeScript("lambda-status=failed");` should be ran inside a bash terminal shell instead of inside added inside the `lambdatest_google_test.js` demo code.

hamishwillee commented on files/en-us/learn/tools_and_testing/cross_browser_testing/your_own_automation_environment/index.md:
```diff
@@ -485,15 +483,15 @@ Let's write an example:
 
 When executing numerous automation tests, marking their status as passed or failed makes the task a lot easier.
```
I'd propose putting these in the example above so that there is no doubt about where they go. Then delete this section and add an explanation below step 5 in the previous section about what this code does. 

But perhaps do it in the other PR so that we're only working on one PR per document.

ShamarYarde replied:
I added it here, because the [refactored code and updated tutorial text to reflect changes](https://github.com/mdn/content/pull/31050#event-11271718332) was merged by the time I got back.

ShamarYarde committed:
moved lines of code to `lambdatest_google_test.js` example

to remove confusion.

ShamarYarde reviewed (commented):
Lines of code moved to `lambdatest_google_test.js` example to remove confusion.

ShamarYarde committed:
formatted code

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

ShamarYarde committed:
Merge branch 'main' into patch-3

ShamarYarde commented:
@hamishwillee I ran out of minutes, so I'm not able to fix the `sauce_google_test.js` example in the Filling in Sauce Labs test details programmatically section, but it looks like it's missing the username in the saucelabs' updateJob method call and the driver.sessionID is undefined.

ShamarYarde commented:
@hamishwillee Can I also put the following change in this pull request? It's only a text change which could be grouped under this pull request's title. It's in number 2 of the second list in the [Your own remote server](https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Cross_browser_testing/Your_own_automation_environment#your_own_remote_server) section.

Description

Updated phrase from "Update the second code block" to "Update the line of code" to reflect changes in the google_test.js example from previous pull requests.

Motivation

To remove potential readers' confusion when looking through google_test.js example for more than one block of code.

ShamarYarde committed:
updated grammar

ShamarYarde committed:
updated grammar and code to reflect changes from previous pull request

Previous text implied there was more than one code block in the `google_test.js` example.

hamishwillee committed:
Merge branch 'main' into patch-3

hamishwillee commented:
@ShamarYarde Sorry for the delay - this wasn't initially assigned to me, and I missed the notification when it was. Will look now.

hamishwillee committed:
Update files/en-us/learn/tools_and_testing/cross_browser_testing/your_own_automation_environment/index.md

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

hamishwillee commented:
@ShamarYarde So what you've done is moved the stuff about marking tests as passed or failed from the separate section `Filling in test details on LambdaTest programmatically` into the body of the main example. Reading that deleted section assumed you'd update the example code, not enter in bash.

That said, I think doing it this way is actually reasonable - there isn't a particular need to highlight this option separately in its own section - in fact by putting it in the body of the example you could argue it makes this something that people are more likely to do by default.

hamishwillee reviewed (approved):
Thanks very much!!!

ShamarYarde commented:
> > I ran out of minutes, so I'm not able to fix the sauce_google_test.js example in the Filling in Sauce Labs test details programmatically section, but it looks like it's missing the username in the saucelabs' updateJob method call and the driver.sessionID is undefined.
> 
> @ShamarYarde I'm going to merge this PR. Do you think you now have bandwidth to look at this case ^^^

@hamishwillee No, I'm still out of minutes. I don't have a subscription.

hamishwillee commented:
Thanks @ShamarYarde - I've created an issue to track the missing bit in #32401 - does that look about right?

As an aside though 
- you know the user name needs to exist in the updateJob method, so wouldn't it be best to update the docs indicating some value, even if we can't verify it? I.e. no worse than it is now.
- You say "driver.sessionID is undefined" - can you expand on what you think needs to happen in the docs?

ShamarYarde commented:
@hamishwillee I don't remember what I saw when I tried it, but it wasn't working. I just looked at the code I have, and the following lines were commented out which means that they were causing the error I was getting:

```
saucelabs.updateJob(username, get_session_id(), {
      name: "Google search results page title test",
      passed: testPassed,
})
```
The response when I run the command: ```node sauce_google_test.js``` now is the following:

```
/.../selenium-test/node_modules/selenium-webdriver/lib/error.js:558
        throw new ctor(message)
              ^

WebDriverError: Uh oh, you've run out of minutes!
Please visit https://saucelabs.com/pricing to purchase a subscription.
    at Object.checkLegacyResponse (/.../Internal/selenium-test/node_modules/selenium-webdriver/lib/error.js:558:15)
    at parseHttpResponse (/.../selenium-test/node_modules/selenium-webdriver/lib/http.js:593:13)
    at Executor.execute (/.../selenium-test/node_modules/selenium-webdriver/lib/http.js:529:28)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  remoteStacktrace: ''
}

Node.js v20.11.0
```

hamishwillee commented:
Thanks!
